src/main/java/edu/kis/powp/command/TriangleFactory.java

```diff
@@ -0,0 +1,29 @@
+package edu.kis.powp.command;
+
+import edu.kis.powp.jobs2d.Job2dDriver;
+import java.util.ArrayList;
+import java.util.List;
+
+public class TriangleFactory {
```

Comment:
You could extract common interface for factories (and use Factory Method pattern)